pkg/apiserver/apiserver.go

```diff
@@ -67,12 +70,15 @@ func (c CompletedConfig) New() (*ServiceCatalogAPIServer, error) {
 	// giving it v1alpha1 version
 	apiGroupInfo.GroupMeta.GroupVersion = v1alpha1.SchemeGroupVersion
 
-	// TODO make storage work
-	//v1alpha1storage := map[string]rest.Storage{}
-	//
-	//v1alpha1storage["servicecatalog"] = apiservice.NewREST(c.RESTOptionsGetter.NewFor(apiregistration.Resource("servicecatalog")))
+	v1Alpha1Storage := map[string]rest.Storage{
```

Comment:
A couple nits and some explanation:

1.  The keys in this map should be lowercase, because they are rest subpaths within the endpoint for this API group
2.  I think we should import the `v1alpha1` package as `servicecatalogv1alpha1`, since there could be multiple API groups in the future
3.  It looks to me in k8s like the convention is to have the following package layout:

```
pkg/
  registry/
    <apigroup>/
      <resource1>/
      <resource2>/
      rest/
        storage_<apigroup>.go
```

...where the storage_*.go for each API group has a method that gives back a `genericapiserver.APIGroupInfo`

So, this stanza of code should move into: `pkg/registry/servicecatalog/rest/storage_servicecatalog.go`.